.gitignore

```diff
@@ -11,25 +12,33 @@
 *.gbc
 *.gb
 
+# for any of the poor souls with save game files in their working directory
+*.sa*
+*.sg*
+*.sn*
+
 # rgbds extras
+*.map
```

Comment:
We have left this out of the .gitignore because we keep .map files in gfx/tilemaps/, gfx/red/, and gfx/blue/
Having this in the .gitignore makes it confusing to edit/rename those files

Reply:
They could be renamed to .tilemap like pokegold and pokecrystal.